orion-proxy/src/xds_configurator.rs

```diff
@@ -114,25 +114,12 @@ impl XdsConfigurationHandler {
         }
     }
 
-    pub async fn xds_run(
```

Comment:
Why this has been removed? It looks like this is used to handle CTRL-C

Reply:
CTRL+C is already handled at line 255 of proxy.rs for the service runtime. That part was a superfluous leftover.

Reply:
Maybe, then it should be removed everywhere else as well... 
I think the root of the problem is not xds but the fact that we drop ***config_senders*** in ***launch_runtimes*** at [45](https://github.com/kmesh-net/orion/blob/main/orion-proxy/src/proxy.rs#L145)

We should return config_senders from those method up to [**run**](https://github.com/kmesh-net/orion/blob/main/orion-proxy/src/lib.rs#L46) and then it seems to be sufficient to add this there :
```
_ = proxy::run_orion(bootstrap, access_logging);
 Ok(())
```